src/Microsoft.AspNet.Http.Extensions/FormFileExtensions.cs

```diff
@@ -0,0 +1,47 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.IO;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.AspNet.Http
+{
+    /// <summary>
+    /// Extension methods for <see cref="IFormFile"/>.
+    /// </summary>
+    public static class FormFileExtensions
+    {
+        private static int DefaultBufferSize = 81920;
```

Comment:
Usually we write these out as multiples of 1024 so it's easier to see the "native" number. And, if possible, a comment indicating why this number was chosen (what's so special about 80KB?).